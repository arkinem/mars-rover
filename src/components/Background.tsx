import React from "react";
import styled from "styled-components";
import Particles, { IParticlesParams } from "react-particles-js";
import { particlesBackgroundConfig } from "../constants/style";

export default () => {
  return (
    <Background
      height={"100%"}
      width={"100%"}
      params={particlesBackgroundConfig as IParticlesParams}
    />
  );
};

const Background = styled(Particles)`
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: -1;
`;
