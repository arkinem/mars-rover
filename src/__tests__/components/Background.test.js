import React from "react";
import renderer from "react-test-renderer";
import Background from "../../components/Background";

describe("Background", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Background />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
