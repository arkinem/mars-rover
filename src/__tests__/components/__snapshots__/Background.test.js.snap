// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background renders correctly 1`] = `
<div
  className="sc-AxjAm hKnwWV"
  id="tsparticles"
>
  <canvas
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</div>
`;
